dnf/yum/config.py

```diff
@@ -738,7 +738,7 @@ def releasever(self, val):
 
     keepcache = BoolOption(False)
     logdir = Option('/var/log') # :api
-    reposdir = ListOption(['/etc/yum/repos.d', '/etc/yum.repos.d']) # :api
+    reposdir = ListOption(['/etc/yum.repos.d', '/etc/yum/repos.d'])  # :api
```

Comment:
Yes, but to prevent inconvenient behavior of installroot it will be wise to applied partial fix for the problem. The final complex fix that will introduce new repository directory with all other steps to keep compatibility (move all files from formal reposdirs to new one, and creation of symlink) can be introduce later.